test/node.js

```diff
@@ -130,7 +131,10 @@ node.onNewRound = function (cb) {
 		if (err) {
 			return cb(err);
 		} else {
-			new Rounds(function (err, rounds) {
+			modulesLoader.initModuleWithDb(Rounds, function (err, rounds) {
+				if (err) {
+					return cb(err);
+				}
 				var nextRound = rounds.calc(height);
```

Comment:
Since `node.js` doesn't implement modules or logic and `rounds.calc()` is more an helper than a module, maybe instead of overload with module inicialization, we can use the plain function from rounds: `Math.ceil(height / slots.delegates)`